test.php

```diff
@@ -2,107 +2,101 @@
 
 namespace App\Importer;
 
-use App\Entity\ClientImporter;
 use App\Entity\FieldConfiguration;
-use App\Model\ImportCriteria;
-use App\Model\SourceField;
 use DateTime;
 use Doctrine\DBAL\Connection;
 use Doctrine\DBAL\DriverManager;
 use Doctrine\DBAL\Query\QueryBuilder;
-use Exception;
 use Google\Cloud\BigQuery\BigQueryClient;
 use MathParser\Exceptions\DivisionByZeroException;
 use MathParser\Interpreting\Evaluator;
-use MathParser\Parsing\Nodes\FunctionNode;
 use MathParser\StdMathParser;
 
 class BaseBigQueryImporter extends BaseImporter
 {
-   private ?Connection $connection = null;
-   private ?BigQueryClient $client = null;
+    private ?Connection $connection = null;
+    private ?BigQueryClient $client = null;
 
-   protected string $tablePrefix = '';
-   protected string $clientKeyField = 'companyName';
-   protected string $dateField = 'date';
-   protected ?string $dateFieldSelect = 'date';
-   protected string $asinField = 'asin';
-   protected ?string $asinFieldSelect = 'asin';
```

Comment:
Remove the null variable because is set as a string and additional we could remove some of these properties because is not used in this class but it depends if in the abstract class is used.
The variables that is not used in this class are:
- $tablePrefix
- $clientKeyField
- $dateFieldSelect
- $asinFieldSelect
- $rootTableAlias